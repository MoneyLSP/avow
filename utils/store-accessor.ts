import { Store } from "vuex"
import { getModule } from "nuxt-property-decorator"
import web3 from "~/store/web3"

// eslint-disable-next-line import/no-mutable-exports
let web3Store: web3

function initialiseStores(store: Store<any>): void {
  // Any additional store modules would go here
  web3Store = getModule(web3, store)
}

export { initialiseStores, web3Store }
