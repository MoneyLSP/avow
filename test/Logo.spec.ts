import Logo from "~/components/AvowLogo.vue"
import "@testing-library/jest-dom"
import { renderWithVuetify as render } from "~/test/utils"

describe("Logo", () => {
  it("should have an image role", () => {
    expect(render(Logo).getByRole("img")).toBeTruthy()
  })
  it("should match a snapshot", () => {
    expect(render(Logo).getByRole("img")).toMatchSnapshot()
  })
})
