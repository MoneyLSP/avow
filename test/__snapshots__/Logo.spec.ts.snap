// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
"<div aria-label=\\"Site Logo\\" role=\\"img\\" class=\\"v-image v-responsive theme--light\\" style=\\"max-width: 35px;\\" id=\\"Logo\\">
  <transition-stub name=\\"fade-transition\\" mode=\\"in-out\\">
    <div class=\\"v-image__image v-image__image--preload v-image__image--cover\\" style=\\"background-position: left center;\\"></div>
  </transition-stub>
  <div class=\\"v-responsive__content\\"></div>
</div>"
`;
